sip-core/src/test/resources/config-test.properties

```diff
@@ -1 +1,3 @@
-sip.core.tracing.limit=5
\ No newline at end of file
+sip.core.tracing.traceType=LOG
```

Comment:
Do we use enum or numeric value in configuration? In sip-core-default-config.yaml default value is numeric.
I prefer enum as it is clearer.

Reply:
Enum, will change yaml